pkg/addons/autopause.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
```

Comment:
2021

also I think the name should be

addons_autopause.go
addons_gcp_auth.go
that way we know these are all the extensions of addons.go and and only inlcudes enable disable code without have to read a lot of other diffferent file names in the file explorer
(but feel free to keep it this way)

Reply:
fair, changed.